include/vcpkg/base/messages.h

```diff
@@ -1067,6 +1075,10 @@ namespace vcpkg
     DECLARE_MESSAGE(ToolFetchFailed, (msg::tool_name), "", "Could not fetch {tool_name}.");
     DECLARE_MESSAGE(ToolInWin10, (), "", "This utility is bundled with Windows 10 or later.");
     DECLARE_MESSAGE(UnexpectedErrorDuringBulkDownload, (), "", "an unexpected error occurred during bulk download.");
+    DECLARE_MESSAGE(UnexpectedFormat,
```

Comment:
I've seen it done for other messages, but I could be wrong. The biggest culprit is when value is used. I just assumed it would match them in order. For this case, I could use expected and actual.